src/battle-log.ts

```diff
@@ -93,16 +99,37 @@ class BattleLog {
 			}
 			break;
 
-		case 'join': case 'j': {
+		case 'join': case 'j': case 'leave': case 'l': {
 			const user = BattleTextParser.parseNameParts(args[1]);
-			divHTML = '<small>' + BattleLog.escapeHTML(user.group + user.name) + ' joined.</small>';
-			break;
-		}
-		case 'leave': case 'l': {
-			const user = BattleTextParser.parseNameParts(args[1]);
-			divHTML = '<small>' + BattleLog.escapeHTML(user.group + user.name) + ' left.</small>';
+			const formattedUser = BattleLog.escapeHTML(user.group + user.name);
+			const isJoin = args[0].includes('j');
```

Comment:
```suggestion
			const isJoin = (args[0].charAt(0) === 'j');
```

d-don't mind me, I'm just golfing.